src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -613,21 +613,13 @@ protected override StatementSyntax CreateAssignmentExpressionStatement(SyntaxTok
                 return SyntaxFactory.ExpressionStatement(CreateAssignmentExpression(identifier, rvalue));
             }
 
-            protected override async Task<StatementSyntax> CreateDeclarationStatementAsync(
+            protected override StatementSyntax CreateDeclarationStatement(
                 VariableInfo variable,
                 ExpressionSyntax initialValue,
                 CancellationToken cancellationToken)
             {
-                // Convert to 'var' if appropriate.  Note: because we're extracting out
-                // to a method, the initialValue will be a method-call.  Types are not
-                // apperant with method calls (i.e. as opposed to a 'new XXX()' expression,
-                // where the type is apperant).
-
-                var options = await SemanticDocument.Document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
no more optoins means this function doesn't have to be async.  that ended up leaking out a lot.